lib/ramble/ramble/experimental/uploader.py

```diff
@@ -304,3 +312,14 @@ def get_experiment_id(experiment):
         # This should be stable per machine/python version, but is not
         # guaranteed to be globally stable
         return hash(json.dumps(experiment, sort_keys=True))
+
+
+class PrintOnlyUploader(Uploader):
+    """An uploader that only prints out formatted data without actually uploading."""
+
+    def perform_upload(self, uri, results):
+        super().perform_upload(uri, results)
+        exp_table_id, exps_to_insert, fom_table_id, foms_to_insert = _prepare_data(results, uri)
+        logger.info("NOTE: The PrintOnly uploader only logs, but does not upload any data.")
+        logger.info(f"Experiments {exps_to_insert} would be uploaded to table {exp_table_id}")
```

Comment:
This would print the whole list of experiments and FOMs right? Is it possible to make it easier to consume this info (like maybe print the length here or at the end, and then print the whole list as one entry per line) ?

I don't know if that's the right choice for this, but it seems like the list being printed in the string would make it hard to connect the different parts.

Reply:
Great call, updated.